common/protos/index.ts

```diff
@@ -35,7 +35,8 @@ export enum VerifyProtoErrorBehaviour {
 export function verifyObjectMatchesProto<Proto>(
   protoType: IProtoClass<any, Proto>,
   object: object,
-  errorBehaviour: VerifyProtoErrorBehaviour = VerifyProtoErrorBehaviour.DEFAULT
+  errorBehaviour: VerifyProtoErrorBehaviour = VerifyProtoErrorBehaviour.DEFAULT,
+  skipUnknownFields = false
```

Comment:
I don't think we should add this parameter - when it is set to true, there is no verification done, so it essentially just does `dataform.dataprep.DataPreparation.create(dataPreparationAsObject)`